Added NVidia GPU Temperature Indicator Applet

sophie-la-li opened the pull request:
As the existing applets regarding NVidia GPU were not working for me (anymore), i added a quick and dirty own one using nvidia-smi as data source.

rcalixte commented:
> As the existing applets regarding NVidia GPU were not working for me (anymore), i added a quick and dirty own one using nvidia-smi as data source.

This will fail validation as-is. In the root of the repository, there is a `validate-spice` script that you can use for local testing.

sophie-la-li commented:
> > As the existing applets regarding NVidia GPU were not working for me (anymore), i added a quick and dirty own one using nvidia-smi as data source.
> 
> This will fail validation as-is. In the root of the repository, there is a `validate-spice` script that you can use for local testing.

Fixed that. Thanks for the hint.

rcalixte commented:
> Fixed that. Thanks for the hint.

One more tip, you should run `cinnamon-spices-makepot` to generate the translation template. You should also consider including the temperature unit (C/F) in some form. Maybe a display option for choosing between the two if you have an interest and/or the time. This is ultimately up to you but it would help disambiguate the interface.

sophie-la-li commented:
> > Fixed that. Thanks for the hint.
> 
> One more tip, you should run `cinnamon-spices-makepot` to generate the translation template. You should also consider including the temperature unit (C/F) in some form. Maybe a display option for choosing between the two if you have an interest and/or the time. This is ultimately up to you but it would help disambiguate the interface.

Did that. I don't have a use for the unit display, but i added it. nvidia-smi always show the temperature in Celsius, so i added a conversion. Hope it's a correct one.

rcalixte commented:
> nvidia-smi always show the temperature in Celsius, so i added a conversion. Hope it's a correct one.

Most temperature-based system tools work in Celsius but there are still some places that insist on not using it. 😉

sophie-la-li commented:
> > nvidia-smi always show the temperature in Celsius, so i added a conversion. Hope it's a correct one.
> 
> Most temperature-based system tools work in Celsius but there are still some places that insist on not using it. 😉

And even there most scientific sectors and actors switched to metric units by now. Fahrenheit is a low tech and consumer unit afaik. But who am i to judge here and now. Thanks for the approval and merge. Have a nice day.
